### PR TITLE
Update 2.5.2

### PR DESCRIPTION
**Pull request by danpetry**
**Destination channel:** defaults

### Links

- [Upstream repository](https://github.com/uclouvain/openjpeg)
- [Upstream changelog/diff](https://github.com/uclouvain/openjpeg/compare/v2.3.0...v2.5.2)
- [conda-forge](https://github.com/conda-forge/openjpeg-feedstock)

### Explanation of changes:

- current openjpeg for windows is [missing a .pc file](https://conda-metadata-app.streamlit.app/?q=pkgs%2Fmain%2Fwin-64%2Fopenjpeg-2.4.0-h4fc8c34_0.conda).
- not needed in the end (for ffmpeg), but did the work so thought maybe it's useful in future?
- Updated to 2.5.2 by pulling conda-forge changes

### CODE REVIEW

**Comment by cbouss** [comment]
We will want to take changes from https://github.com/AnacondaRecipes/openjpeg-feedstock/pull/4 before merging.

**Review by JeanChristopheMorinPerso — APPROVED** [review]
LGTM, but syncing with CF when there is no changes except for the version and build number is probably not worth it considering the noise it creates in the PR IMO.

**Comment by danpetry** [comment]
Screenshot 2024-07-10 at 16 49 30
It made the "commits behind" bit go away, therefore satisfying my ocd

**Comment by JeanChristopheMorinPerso** [comment]
@danpetry All the builds failed.

**Comment by danpetry** [comment]
yeah, I tried adding the tests but it needs [another repo which holds the test artifacts](https://github.com/uclouvain/openjpeg-data). I dropped it due to low priority. Could just leave this PR open for further updates.

File: recipe/bld.bat
Comment (ryanskeith) [inline recipe/bld.bat:15]: As build tests are enabled, does that mean this is packaging them up as well?

**Review by ryanskeith — COMMENTED** [review]
Looks great.  Just a quick question on if we are bundling the tests.

File: recipe/bld.bat
Comment (JeanChristopheMorinPerso) [reply recipe/bld.bat:15]: No. Tests are never installed. Installing stuff in cmake is an explicit action that has to be done in the cmake file.

**Comment by danpetry** [comment]
The tests come from another repo which I need to add but haven't got around to it yet

**Pull request closed without merging**
